site/_includes/partials/devtools/ru/whats-new.md

```diff
@@ -3,20 +3,20 @@
 Чтобы увидеть полный список обновлений,  перейдите на английскую версию по ссылке 
 <a href="/tags/new-in-devtools/" translate="no">What's New In DevTools</a>. Ниже перечислены материалы, переведенные на русский язык.
 
-<!-- ### Chrome 98 {: #chrome98 }
-* [Preview feature: Full-page accessibility tree](/ru/blog/new-in-devtools-98/#a11y-tree)
-* [More precise changes in the Changes tab](/ru/blog/new-in-devtools-98/#changes)
-* [Set longer timeout for user flow recording](/ru/blog/new-in-devtools-98/#recorder-timeout)
-* [Ensure your pages are cacheable with the Back/forward cache tab](/ru/blog/new-in-devtools-98/#bfcache)
-* [New Properties pane filter](/ru/blog/new-in-devtools-98/#properties)
-* [Emulate the CSS forced-colors media feature](/ru/blog/new-in-devtools-98/#forced-colors)
-* [Show rulers on hover command](/ru/blog/new-in-devtools-98/#show-rulers)
-* [Support `row-reverse` and `column-reverse` in the Flexbox editor](/ru/blog/new-in-devtools-98/#flexbox-editor)
-* [New keyboard shortcuts to replay XHR and expand all search results](/ru/blog/new-in-devtools-98/#shortcuts)
-* [Lighthouse 9 in the Lighthouse panel](/ru/blog/new-in-devtools-98/#lighthouse)
-* [Improved Sources panel](/ru/blog/new-in-devtools-98/#sources)
-* [Miscellaneous highlights](/ru/blog/new-in-devtools-98/#misc)
-* [[Experimental] Endpoints in the Reporting API pane](/ru/blog/new-in-devtools-98/#reporting-api) -->
+### Chrome 98 {: #chrome98 }
```

Comment:
Please also make changes in the table of contents (some were made to the headings below)